docpipeline/generate.py

```diff
@@ -118,8 +118,11 @@ def process_blob(blob, credentials, devsite_template):
     )
 
     # Rename the output TOC file to be _toc.yaml to match the expected
-    # format.
-    shutil.move(output_path.joinpath("toc.html"), output_path.joinpath("_toc.yaml"))
+    # format. As well, support both toc.html and toc.yaml
+    try:
+        shutil.move(output_path.joinpath("toc.html"), output_path.joinpath("_toc.yaml"))
+    except FileNotFoundError:
+        shutil.move(output_path.joinpath("toc.yaml"), output_path.joinpath("_toc.yaml"))
```

Comment:
Let's prefer `toc.yaml` over `toc.html`, given that will be more common eventually.

Reply:
Sounds good!